### PR TITLE
BLOB HOTFIX

### PR DESCRIPTION
**Pull request by ChangelingRain**
BLOB CAN ATTACK NOW TM

### CODE REVIEW

**Comment by lzimann** [comment]
:tada:

**Comment by Kromgar** [comment]
You made it incapable of attacking? 10/10

**Comment by ma44** [comment]
when people don't test PRs before merging, this happens

File: code/game/gamemodes/blob/powers.dm [hunk 1]
```diff
@@ -237,7 +237,7 @@
 	expand_blob(T)
 
 /mob/camera/blob/proc/expand_blob(turf/T)
-	if(world.time > last_attack)
+	if(world.time < last_attack)
```
Comment (Jalleo) [inline code/game/gamemodes/blob/powers.dm]: Remember the smaller part is the eyes and the larger part is its mouth. It must want to get hungry at somepoint.

**Pull request merged**